### PR TITLE
Add CIT period tests for packageupgrade test

### PR DESCRIPTION
**Pull request by jjerger**
Adds packageupgrade tests using the staging images for the test repo

### CODE REVIEW

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml [hunk 1]
```diff
@@ -588,4 +610,4 @@ periodics:
...
...imal-1804-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-1804-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2004-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2004-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2204-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2204-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2210-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2210-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2304-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2304-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2310-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2310-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2404-lts-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2404-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2410-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2410-arm64"
```
Comment (a-crate) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml:613]: nit: keep the newline

**Review by a-crate — COMMENTED** [review]
Can you also exclude this from the existing cit-periodics definition?

**Comment by jjerger** [comment]
> Can you also exclude this from the existing cit-periodics definition?

Ah right; done

**Review by a-crate — APPROVED** [review]

**Pull request merged**